platform/.stack-work/dist/aarch64-osx/ghc-9.8.4/stack-project-root

```diff
@@ -0,0 +1 @@
+/Users/chrisaddy/projects/pocketsizefund/pocketsizefund/platform/
```

Comment:
Should this be checked in with your specific local file path reference?

Reply:
nah, this was local cache stuff, I didn't make haskell gitignore, have now fixed

Reply:
fixed